crates/transaction-pool/src/traits.rs

```diff
@@ -613,10 +613,16 @@ impl<'a> CanonicalStateUpdate<'a> {
     }
 }
 
-impl<'a> fmt::Display for CanonicalStateUpdate<'a> {
+impl fmt::Debug for CanonicalStateUpdate<'_> {
```

Comment:
hmm, can we keep Display here?

the debug still might be useful for testing/debugging